BC-Gov-Org-HowTo/Cheatsheet.md

```diff
@@ -0,0 +1,76 @@
+## "Working in github.com/bcgov" Cheatsheet
+
+This github.com/bcgov Cheatsheet covers:
+
+- a few things to help when you are just starting out
+- things to touch base with *every time* you open or contribute to repository (as a BC Government employee)
+
+ All BC Government employees working in github.com/bcgov should be familiar with the full [BC-Policy-Framework-For-GitHub](https://github.com/bcgov/BC-Policy-Framework-For-GitHub)
+
+
+### Joining github.com/bcgov
+- Create a GitHub account with your work email and "Province of British Columbia" as the company (optional) _and_ join the [BCGov GitHub Organization](https://github.com/bcgov)
+- Enable [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) for your GitHub account
```

Comment:
I think 2FA has to be enabled before you can join the bcgov org... suggest moving this bullet before the 'join the bcgov org' bullet